dvc/repo/stage.py

```diff
@@ -473,6 +473,7 @@ def is_out_or_ignored(root, directory):
         for root, dirs, files in self.repo.dvcignore.walk(
             self.fs, self.repo.root_dir
         ):
+            logger.debug(f"Collect Repo Walking {root}")
```

Comment:
This should probably be `trace` level instead of `debug` (so it would be available with `-vv` but not `-v`), as it will generate really verbose output anywhere we are collecting stages (which affects more than just `dvc add`)

Also, even though we use f-strings most of the time, for logging we generally try to use lazy formatting like
```py
logger.trace("Collect repo walking '%s'", root)
```

Reply:
@Erotemic Would you mind addressing this comment?